security/README.md

```diff
@@ -0,0 +1,16 @@
+# Security
+
+## Use case
+This example demonstrates:
+- Vulnerabilities
+- Security Hotspots
+
+## Usage
+
+Run `./run.sh`
+
+This will:
+- Delete the project key **training:security** if it exists in SonarQube (to start from a scratch)
```

Comment:
Is this really necessary to restart from scratch ?
Is this because once a hotspot is "detect"ed or "clear"ed you cannot revert that ?